### PR TITLE
add production user daocloud

### PR DESCRIPTION
**Pull request by allencloud**
Signed-off-by: allencloud <allen.sun@daocloud.io>

add a production user DaoCloud to Documentation/production-users.md

### CODE REVIEW

**Comment by gyuho** [comment]
@allencloud Please fix the commit title according to our contribution guide

`Documentation: add production user daocloud`

Thanks!

File: Documentation/production-users.md [hunk 1]
```diff
@@ -145,3 +145,17 @@ In [hyper.sh][hyper.sh], the container service is backed by [hypernetes][hyperne
...
+
+- *Application*: container management
+- *Launched*: Sep. 2015
+- *Cluster Size*: 1000+ deployments, each deployments contains 3 node cluster.
+- *Order of Data Size*: 100s of Megabytes
+- *Operator*: daocloud.io
+- *Environment*: Baremetal and virtual machines 
+- *Backups*: None, all data can be recreated if necessary.
+
+In [DaoCloud][DaoCloud], we use Docker and Swarm to deploy and run our applications, and use etcd to save meta data for service discovery.
```
Comment (fanminshi) [inline Documentation/production-users.md]: `In [DaoCloud][DaoCloud], we use Docker and Swarm to deploy and run our applications, and use etcd to save meta data for service discovery.` 
=>
`In [DaoCloud][DaoCloud], we use Docker and Swarm to deploy and run our applications, and we use etcd to save metadata for service discovery.`?

**Comment by fanminshi** [comment]
also could you fix the merge conflict? Thanks!

File: Documentation/production-users.md [hunk 2]
```diff
@@ -145,3 +145,17 @@ In [hyper.sh][hyper.sh], the container service is backed by [hypernetes][hyperne
 
 [hypernetes]:https://github.com/hyperhq/hypernetes
 [Hyper.sh]:https://www.hyper.sh
+
+## DaoCloud.io
+
+- *Application*: container management
+- *Launched*: Sep. 2015
+- *Cluster Size*: 1000+ deployments, each deployments contains 3 node cluster.
```
Comment (fanminshi) [inline Documentation/production-users.md]: `each deployments contains 3 node cluster.`
=>
`each deployment contains a 3 node cluster.`?

**Comment by allencloud** [comment]
Thanks a lot for your feedback. Updated now. PTAL @fanminshi

**Comment by fanminshi** [comment]
lgtm. merging.

**Pull request merged**